dunstify.c

```diff
@@ -40,6 +41,7 @@ static GOptionEntry entries[] =
     { "replace",      'r', 0, G_OPTION_ARG_INT,          &replace_id,     "Set id of this notification.", "ID"},
     { "close",        'C', 0, G_OPTION_ARG_INT,          &close_id,       "Close the notification with the specified ID", "ID"},
     { "block",        'b', 0, G_OPTION_ARG_NONE,         &block,          "Block until notification is closed and print close reason", NULL},
+    { G_OPTION_REMAINING, 0, 0, G_OPTION_ARG_FILENAME_ARRAY, &rest, NULL, NULL },
```

Comment:
Just a style note: It isn't aligned to the rest of the options if that is of concern :)

Reply:
Ops my bad. Do you think the other changes are sound?

Reply:
I'm in no position to tell :) Would have to take a deeper look at the dunst code.